packages/cfd/src/Containers/props.types.ts

```diff
@@ -61,3 +62,64 @@ export type TCFDResetPasswordModal = {
     setCFDPasswordResetModal: (value: boolean) => void;
     history: History;
 };
+
+export type TFormValues = {
+    old_password: string;
+    new_password: string;
+    password_type: string;
+};
+
+export type TMultiStepRefProps = {
+    goNextStep: () => void;
+    goPrevStep: () => void;
+};
+
+export type TCountdownComponent = {
+    count_from: number;
+    onTimeout: () => void;
+};
+
+export type TCFDPasswordReset = {
+    sendVerifyEmail: () => Promise<VerifyEmailResponse>;
+    account_type: string;
+    account_group: 'real' | 'demo';
+    server: string;
+    password_type: string;
+};
+
+export type TCFDPasswordManagerTabContentWrapper = {
+    multi_step_ref: React.MutableRefObject<TMultiStepRefProps | undefined>;
+    steps: Array<{ component: JSX.Element }>;
+};
+
+export type TCFDPasswordManagerTabContent = {
+    toggleModal: () => void;
+    selected_login: string;
+    email: string;
+    setPasswordType: (value: string) => void;
+    multi_step_ref: React.MutableRefObject<TMultiStepRefProps | undefined>;
+    platform: 'dxtrade' | 'mt5';
```

Comment:
@hirad-rewok you use repetition in several places (types), can it be better to take it out into a separate type of platform?